### DISCUSSION TITLE
help me

### DISCUSSION THREAD

**Discussion by dohwellington** [Anjok07/ultimatevocalremovergui#864] [General]
I'm on MacOS Sonoma, just updated yesterday. For some reason, I don't know why, but when I try to click the buttons, they are not always responding, regardless if it's my external mouse or touchpad. Can someone please help?

**Comment by dohwellington** [comment]
Unfortunately, following this, I went to install a new update, and now the app is saying it's damaged and must be moved to the trash. What should I do?

**Reply by sdflemming** [reply]
Same here with install attempt on MAC OS 13.5 (22G74). Reinstall the previous UVR5 ( Ultimate Vocal Remover v5.6.0 [2023-10-07 07:22:59] )version is temp solution.
